backend/tests/integration/test_conflict_endpoints.py

```
@@ -0,0 +1,443 @@
+        )
+        test_db_session.add_all([e1, e2])
+        test_db_session.commit()
+
+        response = test_client.get(
+            "/api/events/conflicts",
+            params={"start_date": "2026-06-01", "end_date": "2026-06-30"},
+        )
+        assert response.status_code == 200
+        assert response.json()["conflict_groups"] == []
+
+    def test_time_overlap_detected(self, test_client, overlapping_events):
```

Ruff (0.15.0)

[warning] Unused method argument: `overlapping_events`

(ARG002)